tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -321,9 +321,9 @@ Status ModularFileSystem::IsDirectory(
   return StatusFromTF_Status(plugin_status.get());
 }
 
-Status ModularFileSystem::GetFileSize(
-    const std::string& fname,
-    uint64* file_size /*, TransactionToken* token */) {
+Status ModularFileSystem::GetFileSize(const std::string& fname,
+                                      uint64* file_size,
+                                      TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `file_size` which is output